### PR TITLE
Fixes #29443 - Add Satellite and Capsule 6.8 6.8.z scenarios

### PR DESCRIPTION
**Pull request by upadhyeammit**
Satellite and Capsule 6.8 6.8.z scenarios

### CODE REVIEW

**Comment by kgaikwad** [comment]
> From sat 6.8+ (2.1+) onwards, it will raise out an error.

Correcting myself on this.It will not raise an error but will result in a deprecation warning message.

File: lib/foreman_maintain/concerns/system_helpers.rb [hunk 1]
```diff
@@ -192,6 +192,14 @@ def check_version(name)
           yield current_version
         end
       end
+
+      def installer_args
+        installer_args = ' --disable-system-checks'
```
Comment (kgaikwad) [inline lib/foreman_maintain/concerns/system_helpers.rb]: I don't think this is right place to add change.

File: definitions/features/installer.rb [hunk 2]
```diff
@@ -92,9 +92,7 @@ def run(arguments = '', exec_options = {})
   end
 
   def upgrade(exec_options = {})
-    arguments = '--disable-system-checks'
-    arguments += ' --upgrade' if can_upgrade?
-    run(arguments, exec_options)
+    run(installer_args, exec_options)
```
Comment (kgaikwad) [inline definitions/features/installer.rb]: Two places where we have used `--upgrade` flag. One inside upgrade method of installer feature and second passed as argument to `Procedures::Installer::Run` in package scenario.  

Adding a condition for `--upgrade` flag inside `upgrade` method and changing procedure from `Procedures::Installer::Run` to `Procedures::Installer::Upgrade` inside package scenario won't be sufficient here?

Comment (upadhyeammit) [reply definitions/features/installer.rb]: I moved this as per the suggestion, changed tests accordingly. I am not sure the way I have handled the test change is correct or not, request to review. Thank you !

File: test/definitions/features/installer_test.rb [hunk 3]
```diff
@@ -44,7 +45,7 @@
       it '#upgrade runs the installer with correct params' do
         assume_feature_absent(:satellite)
         installer_inst.expects(:'execute!').
-          with('LANG=en_US.utf-8 foreman-installer --disable-system-checks --upgrade',
+          with("LANG=en_US.utf-8 foreman-installer #{subject.installer_arguments}",
```
Comment (kgaikwad) [inline test/definitions/features/installer_test.rb:48]: Do you really need to call `installer_arguments` method here? 
In before block you are already stubbing this method to return `--disable-system-checks` & `--upgrade` flag.

File: test/definitions/features/installer_test.rb [hunk 4]
```diff
@@ -11,6 +11,7 @@
     before do
       installer_config_dir(["#{data_dir}/installer/simple_config"])
       mock_installer_package('foreman-installer')
+      installer_args
```
Comment (kgaikwad) [inline test/definitions/features/installer_test.rb:14]: You can directly place line here than method.

```suggestion
     Features::Installer.any_instance.stubs(:installer_arguments).returns(' --disable-system-checks \
                                                                         --upgrade')
```

**Comment by kgaikwad** [comment]
Thank you @upadhyeammit!

**Pull request merged**